### DISCUSSION TITLE
Starting out with Home Automation with heyu

### DISCUSSION THREAD

**Discussion by adityadesaha** [HeyuX10Automation/heyu#41] [Q&A] [answered]
Hello people! 

I want to get started with very basic home automation (turning off/on a couple of lamps, fans etc) using my linux computer at home. Is it still a good idea to go with old x10 stuff from ebay/craigslist, or is there a newer standard these days? I'd prefer the old x10 because it's cheap lol.

Also, thank you so much for creating a space for newbies like me to ask stupid questions.

**Comment by shulist** [comment, marked as answer]
X10, while an ancient standard still works fine for simple home automation. As you noted the X10 devices are generally cheap and I've bought some used X10 stuff for dirt cheap (in one case a whole box full). While most X10 devices do not give many options for control they are usually adequate for basic functions (i.e. turn lamp on, adjust brightness, turn lamp off). It is still possible to use them on both Windows 10 and Linux.
